Demo/Demo/ViewModels/PayPalViewModel.swift

```diff
@@ -39,18 +38,18 @@ class PayPalViewModel: ObservableObject {
     }
 
     func checkoutWithNoShipping() {
-        checkout(.noShipping)
+        checkout(ShippingPreference.noShipping)
     }
 
     func checkoutWithProvidedAddress() {
-        checkout(.setProvidedAddress)
+        checkout(ShippingPreference.providedAddress)
     }
 
     func checkoutWithGetFromFile() {
-        checkout(.getFromFile)
+        checkout(ShippingPreference.getFromFile)
     }
```

Comment:
There's likely a simpler more "SwiftUI-y" way to do this selection communication b/w View & ViewModel layers, but that refactor should be a separate PR